data/amplio/units.spec

```diff
@@ -103,7 +104,16 @@ tiles = { "row", "column", "tag"
   2, 15, "u.leader"		; [GB]
   2, 16, "u.barbarian_leader"	; FiftyNine
   2, 17, "u.migrants"		; Eleazar
-;  3, 15, "u.train"		; Eleazar
```

Comment:
It might make sense to move the train to another row and then pop out of this picture the ranks and fuel, like in amplio2.